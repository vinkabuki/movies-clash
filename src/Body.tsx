import React from 'react'

import MoviesCards from './MoviesCards'
import { MoviesDataContext } from './data-context'

const Body: React.FC = () => {
  return <MoviesDataContext.Consumer>{(movies) => <MoviesCards movies={movies} />}</MoviesDataContext.Consumer>
}

export default Body
