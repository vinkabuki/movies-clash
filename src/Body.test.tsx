import React from 'react'
import ShallowRenderer from 'react-test-renderer/shallow'
import Body from './Body'

describe('Body component basic shallow test', () => {
  test('Render Body component', () => {
    const renderer = new ShallowRenderer()
    renderer.render(<Body />)
    const component = renderer.getRenderOutput()
    expect(component).toMatchSnapshot()
  })
})
