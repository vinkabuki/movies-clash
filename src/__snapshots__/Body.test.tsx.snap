// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Body component basic shallow test Render Body component 1`] = `
<Context.Consumer>
  [Function]
</Context.Consumer>
`;
